app/react/portainer/HomeView/EnvironmentList/EnvironmentItem/EnvironmentItem.tsx

```diff
@@ -46,109 +41,94 @@ export function EnvironmentItem({ environment, onClick, groupName }: Props) {
   const route = getRoute(environment);
```

Comment:
Use `getDashboardRoute` from `app/react/portainer/environments/utils/index.ts` and remove `getRoute` function of this file